Gem/Code/Source/SampleComponentManager.cpp

```diff
@@ -1634,4 +1632,31 @@ namespace AtomSampleViewer
             });
     }
 
+    AZ::RHI::MultisampleState SampleComponentManager::CreateDefaultMsaaState() const
+    {
+        AZ_Assert(IsValidNumMSAASamples(m_numMSAASamples), "Invalid MSAA sample setting");
+
+        AZ::RHI::MultisampleState state;
+        state.m_samples = static_cast<uint16_t>(m_numMSAASamples);
+        state.m_customPositionsCount = m_numMSAASamples;
+        state.m_customPositions[0] = { 8, 8 };  
```

Comment:
Add a comment about the custom positions. Would be nice to see a diagram of it.

Reply:
Added a comment. Good idea since our notation is different from hlsl

Reply:
Nit. Maybe add a line saying 

"We are filling all the possible positions here even though the positions taken into consideration will be capped by m_customPositionsCount"